### DISCUSSION TITLE
Change amount of claimblocks per purchase?

### DISCUSSION THREAD

**Discussion by foxotic** [GriefPrevention/GriefPrevention#918] [Q&A] [answered]
Instead of 1 block per purchase, how can I set it to do 100 (10x10) claimblocks per 15 points (using Gringott's to be 15 emeralds per purchase)

**Comment by RoboMWM** [comment]
100 / 15 = 6.6 repeating, so 6.7 rounded, or 7 for an integer.

**Comment by foxotic** [comment]
Robo wait, this isn't what I'm asking. Instead of calculating the price to "100" blocks, can you please set an option to ONLY buy 100. example /buyclaimblocks 1 = 100 blocks. /buyclaimblocks 2 = 200

**Comment by RoboMWM** [comment, marked as answer]
No, not possible in the built-in implementation.

An addon can provide this feature - use the feature request template if there isn't already a request for this out there.
